packages/homepage/package.json

```diff
@@ -55,7 +55,7 @@
     "gatsby-react-router-scroll": "^2.1.3",
     "gatsby-remark-autolink-headers": "^2.1.3",
     "gatsby-remark-copy-linked-files": "^2.1.4",
-    "gatsby-remark-embedder": "^1.1.2",
+    "gatsby-remark-embedder": "^4.2.0",
```

Comment:
These updates are really necessary to make this PR works? I wouldn't encourage update packages, since they might come with unexpected behaviors or bugs.

Reply:
The only "breaking change" here is [`v2.0.0`](https://github.com/MichaelDeBoey/gatsby-remark-embedder/releases/tag/v2.0.0), which just requires `node@>=10.13.0` & 
`gatsby@^2.20.0`

All other breaking changes are not applicable for this repo (see [releases](https://github.com/MichaelDeBoey/gatsby-remark-embedder/releases) page)